core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20211115_v1_workspace_migration_history.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="ehigham" id="create_V1_WORKSPACE_MIGRATION_HISTORY">
+        <createTable tableName="V1_WORKSPACE_MIGRATION_HISTORY"
+                     remarks="Records the history and progress of v1 -> v2 workspace migration attempts">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="WORKSPACE_ID"
```

Comment:
Can we make this a FK to the workspace table and set it to cascade delete?

Reply:
This workspace id is not unique in this table. Might that be a problem?

Reply:
done